frontend/src/component/PublicProfile/CustomFormManager.jsx

```diff
@@ -19,19 +19,20 @@ const DisplayTitle = props => {
   )
 }
 
+const labelComponent = props => (
```

Comment:
turn it to LabelComponent so you can call it like so `<LabelComponent {...props} />` instead of `{labelComponent(props)}`

you can even add the `translate()` HOC to avoid passing the `t={props.t}` everywhere

Reply:
done, thanks